plugins/trendmicro_visionone/plugin.spec.yaml

```
@@ -2,13 +2,18 @@ plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
 name: trendmicro_visionone
-title: Trend Micro Vision One
-description: Trend Micro Vision One is an enhanced threat defense platform, surpassing standard XDR solutions. It offers comprehensive detection and response across various security layers and automates data correlation for rapid response, effectively preventing most attacks
-version: 4.0.0
-connection_version: 3
-supported_versions: [Trend Micro Vision One API v3]
+title: Trend Vision One
+description: Trend Vision One is an enhanced threat defense platform, surpassing standard XDR solutions. It offers comprehensive detection and response across various security layers and automates data correlation for rapid response, effectively preventing most attacks
+version: 5.0.0
+connection_version: 4
```

If major 5 then update conn version to 5
```suggestion
connection_version: 5
```